File: ProjectFinalAdminBD/src/java/dominio/Color.java
package dominio;

/**
 * @author emfam
 */
public class Color {
    private int idColor;
    private String color;
    
    public Color(){}

    public Color(int idColor, String color) {
        this.idColor = idColor;
        this.color = color;
    }

    public int getIdColor() {
        return idColor;
    }

    public void setIdColor(int idColor) {
        this.idColor = idColor;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    @Override
    public String toString() {
        return "Color{" + "idColor=" + idColor + ", color=" + color + '}';
    }
    
    
}

File: ProyectoBD2021/domain/Color.php
<?php
	
	class Color
	{

		private $idColor;
		private $color;

		function __construct($idColor,$color)
		{
			$this->idColor = $idColor;
			$this->color = $color;
		}


		public function getIdColor()
		{
			return $this->idColor;
		}

		public function getColor()
		{
			return $this->color;
		}

	}
 ?>